hugo/assets/ts/helpers/cookie-service.ts

```diff
@@ -1,15 +1,36 @@
+export enum CookieCategory {
+    NECESSARY = 'necessary',
+    PREFERENCES = 'preferences',
+    STATISTICS = 'statistics',
+    MARKETING = 'marketing'
+}
+
+interface CookieWall {
+    hasConsent(category: CookieCategory): boolean;
+    showCookieChoice(): void;
+    showPreferences(): void;
+}
+
 export class CookieService {
-    static hasAcceptedCookies(): boolean {
-        return (CookieService.getCookie('cue-cookies') === 'yes');
+    static hasConsent(category: CookieCategory): boolean {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        return !(window as Window).cookiewall ||
+            ((window as Window).cookiewall as CookieWall)?.hasConsent(category);
+        // return (CookieService.getCookie('cue-cookies') === 'yes');
```

Comment:
@JorindeUsMedia didn't really know what to do with this part since there is no cookiewall yet. Do you have any ideas on how to set this up for now?

Reply:
will depend a lot on which service we're going to use. The window.cookiewall in this case is our own cookiewall. Most cookieservices work with getting the cookiechoice from a cookie so maybe prepare something for that. So remove the window.cookiewall stuff but keep the category param (maybe add a _ because we're not using it now) and for now just let the cookieservice return false

Reply:
or.. we can implement our own cookiewall for now... which would also be an option.